TrkReco/inc/TrkTimeCalculator.hh

```diff
@@ -29,33 +29,31 @@ namespace mu2e
        {
           using Name    = fhicl::Name;
           using Comment = fhicl::Comment;
-          fhicl::Atom<int>    debug{        Name("debugLevel"),          Comment("Debug flag"),0 }; 
-          //fhicl::Atom<bool>   useFlag{      Name("useFlag"),      Comment("Which flags to use") }; 
-          //fhicl::Atom<bool>   dontUseFlag{  Name("dontUseFlag"),  Comment("Which flags to skip") }; 
-          fhicl::Atom<double> avgDriftTime{ Name("AverageDriftTime"),    Comment("Average time offset for straw hits"),22.5 }; 
-          fhicl::Atom<bool>   useTOTdrift{  Name("UseTOTDrift"),         Comment("Switch to use the time drift"),true }; 
-          fhicl::Atom<double> beta{         Name("StrawHitBeta"),        Comment("Beta of the particle-hypothesis used"),1.0 }; 
-          fhicl::Atom<double> shErr{        Name("StrawHitTimeErr"),     Comment("Effective hit time res. without TOT"),9.7 }; 
-          fhicl::Atom<double> caloZOffset{  Name("TrkToCaloTimeOffset"), Comment("Location of shower max WRT COG Z position"),-120.0 }; 
-          fhicl::Atom<double> caloT0Offset{ Name("CaloTimeErr"),         Comment("Time offsets for downstream particls in the calorimeter"),-0.4 }; 
-          fhicl::Atom<double> caloT0Err{    Name("CaloT0Err"),    Comment("Time resolution for calorimeter"),0.5 }; 
-       };       
+          fhicl::Atom<int>     debug{          Name("debugLevel"),          Comment("Debug flag"),0 };
+          fhicl::Atom<double>  avgDriftTime{   Name("AverageDriftTime"),    Comment("Average time offset for straw hits"),22.5 };
+          fhicl::Atom<bool>    useTOTdrift{    Name("UseTOTDrift"),         Comment("Switch to use the time drift"),true };
+          fhicl::Atom<double>  beta{           Name("StrawHitBeta"),        Comment("Beta of the particle-hypothesis used"),1.0 };
+          fhicl::Atom<double>  shErr{          Name("StrawHitTimeErr"),     Comment("Effective hit time res. without TOT"),9.7 };
+          fhicl::Atom<double>  caloTimeOffset{ Name("TrkToCaloTimeOffset"), Comment("Time offsets for downstream particls in the calorimeter"),-0.4 };
+          fhicl::Atom<double>  caloZOffset{    Name("CaloShowermaxZ"),      Comment("Location of shower max WRT COG Z position"),-120.0 };
```

Comment:
typo in the help text "particls" missing "e"

Reply:
fixed.